demo/src/screens/realExamples/ProductPage/index.tsx

```diff
@@ -0,0 +1,138 @@
+        </View>
+    
+        <View paddingH-page>
+          <View row centerV>
+            <Text text40M marginT-s7>New Product</Text>
+          </View>
+          <Text text80 dark40 marginV-s1>SKU: 1234567890</Text>
+          <Text text50L marginV-s2>$55.00</Text>
+      
+          <View marginV-s2 paddingV-s2 style={{borderColor: Colors.dark70, borderBottomWidth: 1}}>
+            <Text dark30>Color</Text>
+            <TouchableOpacity row centerV spread onPress={()=> this.setState({isColor: true})}>
+              <View row centerV>
+                <View style={{width: 14, height: 14, borderRadius: 12, marginRight: 6}} backgroundColor={colorOption.color}/>
+                <Text dark10 text70>{colorOption.name}</Text>
+              </View>
+              <Image source={require('../../../assets/icons/chevronDown.png')}/>
+            </TouchableOpacity>
+          </View>
+          <ActionSheet
+            title={'Select a Color'}
+            options={[
+              {label: colorOptions.red.name, onPress: () => this.pickOption(colorOptions.red)},
+              {label: colorOptions.green.name, onPress: () => this.pickOption(colorOptions.green)},
+              {label: colorOptions.blue.name, onPress: () => this.pickOption(colorOptions.blue)},
+            ]}
+            visible={isColor}
+            onDismiss={()=> this.setState({isColor: false})}
+          />
+      
+          <View marginV-s2 paddingV-s2 style={{borderColor: Colors.dark70, borderBottomWidth: 1}}>
+            <Text dark30>Size</Text>
+            <TouchableOpacity row centerV spread onPress={()=> this.setState({isSize: true})}>
+              <View row centerV>
+                <Text dark10 text70>{sizeOption.name}</Text>
+              </View>
+              <Image source={require('../../../assets/icons/chevronDown.png')}/>
+            </TouchableOpacity>
+          </View>
+          <ActionSheet
+            title={'Select a Size'}
+            options={[
+              {label: sizeOptions.s.name, onPress: () => this.pickOption(sizeOptions.s)},
+              {label: sizeOptions.m.name, onPress: () => this.pickOption(sizeOptions.m)},
+              {label: sizeOptions.l.name, onPress: () => this.pickOption(sizeOptions.l)},
+            ]}
+            visible={isSize}
+            onDismiss={()=> this.setState({isSize: false})}
+          />
+      
+          <Button style={{marginVertical: 40}} label={'Add to Cart'} />
+      
+          <Text text60M>Recommended for You</Text>
+        </View>
+        <ScrollView style={{height: 200, marginVertical: 20, marginLeft: 20}} horizontal showsHorizontalScrollIndicator={false}>
+          {
+            images.map((image) => (
+              <Image
+                width={160}
+                style={{marginRight: 10}}
+                source={{uri: image}}
+              />
+            ))
+          }
+        </ScrollView>
+      </ScrollView>
+    );
+  }
+}
+
+export default Product;
```

Comment:
Missing empty space at the end
```suggestion
export default Product;

```